addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -984,6 +985,18 @@ var SnippetsMenu = Widget.extend({
             }, 1000);
         });
     },
+    /**
+     * @override
+     */
+    async willStart() {
+        await this._super(...arguments);
+        const users = await this._rpc({
+            model: 'res.users',
+            method: 'read',
+            args: [[session.user_id], ['lang']],
+        });
+        this._userLanguage = users[0].lang;
+    },
```

Comment:
Why changing this https://github.com/odoo/odoo/blob/13.0/addons/web/models/ir_http.py#L71 instead would not work?